app/assets/stylesheets/custom.scss

```diff
@@ -184,11 +184,26 @@ a {
 
 .listing-slide {
   margin-top: 20px;
-  margin-bottom: 50px;
+  margin-bottom: 20px;
   margin-left: 20px;
   margin-right: 20px;
 }
 
+.wrapper {
+  text-align: center;
+}
+
+.mappy {
+  float: right;
+  width: 600px;
```

Comment:
Properties should be ordered float, min-height, padding-left, padding-right, padding-top, top, width